konfera/fixtures/test_data.json

```diff
@@ -4,7 +4,7 @@
   "pk": 1,
   "fields": {
     "uuid": "dc81d075-e7ba-42ae-a081-1c0a6bf935c3",
-    "date_created": "2016-10-01T18:00:00Z",
+    "date_created": "2016-10-23T21:27:42.090Z",
     "date_modified": "2016-10-01T18:00:00Z",
```

Comment:
Hmm... I think something is not okay with the model itself.

Why did the `date_created` changed instead of `date_modified`?